### ISSUE TITLE
ocr crnn 训练时出错

### ISSUE THREAD

**Issue by wuxiaolianggit** [chineseocr/chineseocr#221]
train_loader = torch.utils.data.DataLoader(
    traindataset, batch_size=batchSize,
    shuffle=False, sampler=None,
    num_workers=int(workers),
    collate_fn=alignCollate(imgH=imgH, imgW=imgW, keep_ratio=keep_ratio))

train_iter = iter(train_loader)

len(train_iter) 一直等于0，请问时怎么回事呢？

**Comment by Zweo** [comment]
能顺便问您一下，warpctc_pytorch 这个包是怎么导入的呢？准备训练的时候，这个包一直编译不成功

**Comment by Aurora11111** [comment]
@Zweo reference:
https://github.com/Aurora11111/crnn-train-pytorch

**Comment by Zweo** [comment]
> @Zweo reference:
> https://github.com/Aurora11111/crnn-train-pytorch

Thanks

**Comment by Zweo** [comment]
@Aurora11111  thanks

**Comment by wenlihaoyu** [comment]
warpctc_pytorch 是一个扩展包 ，安装参考https://github.com/SeanNaren/warp-ctc

**Comment by jackstraw2012** [comment]
@wuxiaolianggit 请教一下数据标注和训练的问题。希望能加我QQ 237080818，万分感谢。
